analyzer.go

```diff
@@ -1,50 +1,52 @@
 package lifecycle
 
 import (
-	"fmt"
-	"path/filepath"
-
 	"github.com/buildpacks/imgutil"
 	"github.com/pkg/errors"
 
+	"github.com/buildpacks/lifecycle/api"
 	"github.com/buildpacks/lifecycle/buildpack"
-	"github.com/buildpacks/lifecycle/launch"
+	"github.com/buildpacks/lifecycle/cmd"
```

Comment:
I think we should just copy the `Platform` interface, rather than introduce a dependency on `cmd`.

Reply:
mmmmm...I don't like copying the interface multiple times although I know it's common in Golang.
Maybe we can put it in a separate file/package that we can use everywhere?

Reply:
I don't have a strong opinion here on if we move it or copy. Will you 2 get together and come back with a recommendation?

Reply:
We talked it over and we're leaning towards copy because the lifecycle package only needs 

```
type Platform interface {
	API() string
}
```

(not `CodeFor()`).

Reply:
As @natalieparellano wrote, let's copy it for now and if we think about a better solution, we'll change it in a separate PR.